main.cc

```diff
@@ -402,6 +402,8 @@ int main( int argc, char ** argv )
   app.setStyle(new GdAppStyle);
 #endif
 
+  app.setDesktopFileName( "org.goldendict.GoldenDict.desktop" );
```

Comment:
[Wrap in](https://doc.qt.io/qt-5/qguiapplication.html#desktopFileName-prop) `#if QT_VERSION >= QT_VERSION_CHECK( 5, 7, 0 )` and wrap the string literal in `QStringLiteral( )`, which is available since Qt 5.